src/library/styles/__tests__/pxToEm.spec.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+import { pxToEm } from '../../styles';
+
+describe('pxToEm', () => {
+  describe('with string input', () => {
+    it('works without units', () => {
+      expect(pxToEm('2')).toEqual('0.125em');
+    });
+
+    it('works with px units', () => {
+      expect(pxToEm('2px')).toEqual('0.125em');
+    });
+
+    it('disregards incoming non-px units', () => {
+      expect(pxToEm('2em')).toEqual('2em');
+      expect(pxToEm('2rem')).toEqual('2rem');
+      expect(pxToEm('2anything')).toEqual('2anything');
+    });
+
+    it('respects floats', () => {
+      expect(pxToEm('2.5px')).toEqual('0.15625em');
+    });
+  });
+
+  describe('with number input', () => {
+    it('works', () => {
+      expect(pxToEm(2)).toEqual('0.125em');
+    });
+
+    it('respects floats', () => {
+      expect(pxToEm(2.5)).toEqual('0.15625em');
+    });
+  });
+
+  describe('with other input types', () => {
+    it('errors', () => {
+      // $FlowFixMe
```

Comment:
You should probably add a brief explanation (e.g. "We want this error") as to why these are necessary.